CMakeLists.txt

```diff
@@ -74,6 +74,16 @@ option(ENABLE_MULTI_PROCESS_BUILDS "Enable /MP flag for MSVS solution" ON)
 option(COPY_CONFIG_ON_BUILD "Copies config folder into output directory at building phase" ON)
 option(ENABLE_STATIC_AI_LIBS "Add AI code into VCMI lib directly" ${staticAI})
 
+option(ENABLE_COLORIZED_COMPILER_OUTPUT "Colorize compilation output (Clang/GNU)." OFF)
```

Comment:
why not set it to ON by default?

Reply:
Just precaution, I'm not sure if there are any negative implications to enabling this? There could be, I just don't know. I don't mind setting it to ON by default and should someone wind up with problems they can file an issue.

Reply:
Changed default to `ON` with [4f4298e](https://github.com/vcmi/vcmi/pull/3422/commits/4f4298ece872981fd54e7119991dd87ac113b561)